unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionDeployedScenesController.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using DCL.Helpers;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+internal class SectionDeployedScenesController : SectionBase, IDeployedSceneListener, ISectionHideContextMenuRequester
+{
+    public const string VIEW_PREFAB_PATH = "BuilderProjectsPanelMenuSections/SectionDeployedScenesView";
+    
+    public event Action OnRequestContextMenuHide;
+    
+    public override ISectionSearchHandler searchHandler => sceneSearchHandler;
+
+    private readonly SectionDeployedScenesView view;
+
+    private readonly SceneSearchHandler sceneSearchHandler = new SceneSearchHandler();
+    private Dictionary<string, SceneCardView> scenesViews;
+
+    public SectionDeployedScenesController(): this(
+        Object.Instantiate(Resources.Load<SectionDeployedScenesView>(VIEW_PREFAB_PATH))
+    )
+    {
+    }
+    
+    public SectionDeployedScenesController(SectionDeployedScenesView view)
```

Comment:
Only as suggestion: It would be a good practice to use here an `ISectionDeployedScenesView` interface in order to be able to pass a mocked view and ease the tests for the `SectionDeployedScenesController` controller.

It would let you to completely separate the tests for `SectionDeployedScenesController` and for `SectionDeployedScenesView`.

Reply:
I use SectionDeployedScenesView with methods and events of the class without accessing directly any method or property from the monobehavior to make it easy to replace it by an interface if needed. Since I'm not using NSubstitute to mock the view because I was aiming for a more integrated these tests I feel is better not to add unnecessarily and interface until we feel that we need it to extend the tests in the future